### ISSUE TITLE
Don't force lowercase for bounty/claim names and descriptions

### ISSUE THREAD

**Issue by picsoritdidnthappen** [picsoritdidnthappen/poidh-app#114]
Certain URLs require that uppercase/lowercase URLs be understandable. For example, I was not able to check this google docs link because the app made all uppercase letters lowercase:

<img width="511" alt="Screen Shot 2024-05-16 at 8 55 19 AM" src="https://github.com/picsoritdidnthappen/poidh-app/assets/152807131/85c65438-0e07-4d74-ae25-d69346f354e7">

We need to ensure that the "force lowercase" rule DOES NOT apply to bounty claim NFT names or descriptions. "force lowercase" should also NOT apply to overall bounty names and descriptions

Basically, if the data comes from a user input, lowercase should not be forced

**Comment by restubilal** [comment]
https://github.com/picsoritdidnthappen/poidh-app/blob/8f93ac9c5a6f0de0e77bde1816b2efee589d4790/src/components/bounty/ProofItem.tsx#L142


edit line 144 and 145 

<p className='' style={{ textTransform: 'none' }}>{title}</p>
          <p className='break-all' style={{ textTransform: 'none' }}>{description}</p>

**Comment by ie2173** [comment]
Pull Request Created
